tests/readme.md

```diff
@@ -26,3 +26,27 @@ This starts the following Docker containers and runs the test cases against them
 * Hono MQTT adapter
 * Hono AMQP adapter
 * Hono CoAP adapter
+
+To run a single test, set the `it.test` property:
+
+    $ mvn verify -Prun-tests -Dit.test=TelemetryHttpIT
+
+The `logging.profile` property with a value of either `prod`, `dev` or `trace` can be used to set the log level in the Hono Docker containers:
+
+    $ mvn verify -Prun-tests -Dlogging.profile=trace
+
+### Running the Tests without starting/stopping the containers
+
+When running the tests with the `docker.keepRunning` property, the Docker containers will not be stopped and removed once the tests are complete:
+
+    $ mvn verify -Prun-tests -Ddocker.keepRunning
+
+Subsequent test runs can use the running containers and will thereby finish much faster by adding the `useRunningContainers` profile to the maven command:
+
+    $ mvn verify -Prun-tests,useRunningContainers
```

Comment:
I assume I can also combine this with `-Dit.test=`, right?

Reply:
Yes.